db/schema.rb

```diff
@@ -31,9 +52,13 @@
     t.string "encrypted_password", limit: 128, null: false
     t.string "confirmation_token", limit: 128
     t.string "remember_token", limit: 128, null: false
+    t.string "name"
+    t.string "Description"
+    t.string "description"
```

Comment:
Why two descriptions on the same table? Just asking

Reply:
why this?